outputs.tf

```diff
@@ -1,4 +1,52 @@
-# output "variable" {
-#   description = "output variable description"
-#   value       = variable.main.name
-# }
+output "main_host_id" {
```

Comment:
I'm not sure if `main_` and `redis_` prefixes are needed in outputs. Please rethink this.